Sources/PrintingKit/PrintingKit.docc/Getting-Started.md

```diff
@@ -60,3 +60,10 @@ This will show a print dialog that you can use to perform the print operation, o
 ## PDF support
 
 PrintingKit also has ``Pdf`` utilities, that can be used to configure a PDF documentations, page margins, etc. They're used by the SDK when printing ``PrintItem/pdfData(_:)`` and ``PrintItem/pdfFile(at:)``, but you can use then as standalone utilities as well.
+
+
+## Configuring your project for macOS
+
+There are two settings that are manadatory for a sandboxed application on macOS (which is default). In the target's "Signing & Capabilities" > "App Sandbox" section 
```

Comment:
I think we have to add a line break after the first paragraph to automatically make the numbered list